api/src/core/decorators/user-id.decorator.ts

```diff
@@ -1,11 +1,10 @@
 import { type ExecutionContext, InternalServerErrorException, createParamDecorator } from '@nestjs/common';
 import { type Request } from 'express';
-import { Types, isValidObjectId } from 'mongoose';
 
 export const UserId = createParamDecorator((_, context: ExecutionContext) => {
   const request = context.switchToHttp().getRequest<Request>();
-  if (!isValidObjectId(request.user?.id)) {
+  if (!request.user?.id) {
     throw new InternalServerErrorException(`Value cannot be coerced to object ID: ${request.user?.id}`);
```

Comment:
This error message doesn't make that much sense anymore, since the only reason this would be called is if it it is undefined or empty.

Reply:
Also, see if you can use the shared implementation:

```typescript
import { CurrentUser } from '@douglasneuroinformatics/nestjs/core';
```